modules/rte_mis_student_tracking/rte_mis_student_tracking.module

```diff
@@ -24,3 +28,185 @@ function rte_mis_student_tracking_get_allowed_values(FieldStorageDefinitionInter
   $options = rte_mis_core_get_allowed_values($definition, $entity);
   return $options;
 }
+
+/**
+ * Implements hook_form_alter().
+ */
+function rte_mis_student_tracking_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  // Check if this is the exposed filter form you want to modify.
+  if ($form_id == 'views_exposed_form' && isset($form['#id']) && $form['#id'] == 'views-exposed-form-student-tracking-dashboard-page-1') {
+
+    $options = rte_mis_student_tracking_class_list_options();
+
+    // Add a checkboxes element to the form.
+    $form['class_filter'] = [
+      '#type' => 'checkboxes',
+      '#options' => $options,
+      '#default_value' => [],
+    ];
+
+    // Create a list of values for total students.
+    $items = [];
+    foreach (rte_mis_student_tracking_get_total_students() as $value) {
+      $items[] = $value;
+    }
+
+    // Add the list of values as a markup element.
+    $form['total_students'] = [
+      '#type' => 'markup',
+      '#markup' => '<ul><li>' . implode('</li><li>', $items) . '</li></ul>',
+    ];
+
+    $form['#cache'] = [
+      'tags' => [
+        'config:rte_mis_student_tracking.settings',
+        'config:rte_mis_school.settings',
+      ],
```

Comment:
Please fetch the cache tag from config object.

Reply:
ok